simpledotorg/simple-server
# Wire up progress tab v2

**Story card:** [sc-9020](https://app.shortcut.com/simpledotorg/story/9020/wire-up-the-progress-tab-v2)

## Because

The views were broken

## This addresses

Add a FakeProgressService to get an idea of the structure of the data required to render the views

## Other changes

- Extract some partials into the view component
- Re-organise progress tab-related files 

## Test instructions

- Check out this branch
- Make sure `new_progress_tab_v2` is enabled
- goto `http://localhost:3000/api/v3/analytics/user_analytics.html`. Use ModHeader or Postman to add necessary headers
- See more details [here](https://github.com/simpledotorg/simple-server#testing-web-views)
- You should be able to see the progress tab reports
